javascript/conditonal-statement.md

```diff
@@ -0,0 +1,259 @@
+# CONDITIONAL STATEMENT IN JAVASCRIPT.
+
+Conditional statements are statements that perform particular tasks based on the given condition(s).
+
+## This project explains the following:
+
+- What conditional statements are.
+- The `if` statement.
+- The `if...else` statement.
+- Logical operators.
+- ternary operators.
+- Truthy and falsy.
+- Truthy and falsy assignment.
```

Comment:
Is there a significant difference between these two? I think merging them as one would be better in this case. What do you think? 🤔 

_Please do this for all other occurrences._